index.html

```diff
@@ -576,355 +533,6 @@ <h3>
           <img src="timestamp-diagram.svg" alt="Navigation Timing attributes"
           style='width: 100%'>
         </figure>
-        <ol>
-          <li>If the navigation is aborted for any of the following reasons,
```

Comment:
Is this covered by the HTML PR?

Reply:
Yes, there will be no load info for an aborted document